### PR TITLE
Grooveretirement

### PR DESCRIPTION
**Pull request by sakley**
Update doc for Groove retirement

### CODE REVIEW

**Comment by sakley** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C3%5Caf56be8f-c68d-4cc9-d622-923aca40b12a%5CPullRequest%5C201710030829270153-37%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=alR8tpRBbQJ1X%2B%2F2taK9w%2F%2FBnVSX%2F0fOmbwcgeAIzdo%3D&st=2017-10-03T08%3A25%3A26Z&se=2017-10-04T08%3A35%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sakley** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Contribute.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/Contribute.md) | :white_check_mark:Succeeded | |
[README.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/README.md) | :white_check_mark:Succeeded | |
[groove/api-overview.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/api-overview.md) | :white_check_mark:Succeeded | |
[groove/faq.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/faq.md) | :white_check_mark:Succeeded | |
[groove/getting-started.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/getting-started.md) | :white_check_mark:Succeeded | |
[groove/groove-api-terms-of-use.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-api-terms-of-use.md) | :white_check_mark:Succeeded | |
[groove/groove-service-rest-reference/common-parameters.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-service-rest-reference/common-parameters.md) | :white_check_mark:Succeeded | |
[groove/groove-service-rest-reference/extras.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-service-rest-reference/extras.md) | :white_check_mark:Succeeded | |
[groove/groove-service-rest-reference/http-status-codes.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-service-rest-reference/http-status-codes.md) | :white_check_mark:Succeeded | |
[groove/groove-service-rest-reference/namespace.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-service-rest-reference/namespace.md) | :white_check_mark:Succeeded | |
[groove/groove-service-rest-reference/orderby.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-service-rest-reference/orderby.md) | :white_check_mark:Succeeded | |
[groove/groove-service-rest-reference/overview.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/groove-service-rest-reference/overview.md) | :white_check_mark:Succeeded | |
[groove/sdk-and-helpers/sdk-list.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/sdk-and-helpers/sdk-list.md) | :white_check_mark:Succeeded | |
[groove/support.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/support.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/Application-Registration.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/Application-Registration.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/guidelines.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/guidelines.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/obtaining-a-developer-access-token-legacy.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/obtaining-a-developer-access-token-legacy.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/obtaining-a-developer-access-token.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/obtaining-a-developer-access-token.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/overview.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/overview.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/streaming-music.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/streaming-music.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/user-authentication-in-applications.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/user-authentication-in-applications.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/user-authentication-on-the-web.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/user-authentication-on-the-web.md) | :white_check_mark:Succeeded | |
[groove/using-the-groove-restful-services/user-authentication.md](https://github.com/MicrosoftDocs/groove-api-documentation/blob/GrooveRetirement/groove/using-the-groove-restful-services/user-authentication.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C3%5Caf56be8f-c68d-4cc9-d622-923aca40b12a%5CPullRequest%5C201710030831159473-37%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JP0uFs%2Flyer5Vh5OJXFAXUKH6cTQC%2BnDDLQSV53MszE%3D&st=2017-10-03T08%3A27%3A06Z&se=2017-10-04T08%3A37%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: groove/faq.md [hunk 1]
```diff
@@ -25,6 +26,12 @@ ms.assetid: efff8209-3f96-786d-0bec-bdd942294238
 - [May I download audio files with my application and provide a download feature to my application users?](#18)
 - [Where did the Pilot program go?](#20)
 
+
+### <a name="0"> </a>I want to develop an app with the Groove API but it seems the Sign-up process is not working anymore, what is going on?
+The Groove Music API programme is no longer onboarding any new developper or app.
+Please check [https://aka.ms/grooveCRMfaq]
```
Comment (daauro) [inline groove/faq.md]: >grooveCRMfaq [](start = 29, length = 12)

groovePartnerFaq?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+| Notice to customers|
+|----- |
+|Starting Oct 2nd, the Onboarding to the Groove Music API is disabled. As part of the partnership, the Groove Music Pass service will be discontinued on December 31, 2017.��
```
Comment (bfreydier) [inline README.md]: >Onboarding [](start = 23, length = 10)

nit: onboarding

File: groove/api-overview.md [hunk 3]
```diff
@@ -6,6 +6,12 @@ author: sakley
 ms.assetid: 53da26dd-2573-8b1a-0454-821687f079cf
 ---
 
+| Notice to customers|
+|----- |
+|Starting Oct 2nd, the Onboarding to the Groove Music API is disabled. As part of the partnership, the Groove Music Pass service will be discontinued on December 31, 2017.��
+After that date, Groove Music Pass content will not stream or play and our API features will not be accessible.
+Please check our FAQ on <https://aka.ms/groovepartnerfaq> . All features of the Music API will be supported until Dec 31st.|
+
```
Comment (daauro) [inline groove/api-overview.md]: I'm sure this can be factorized

**Pull request merged**